initscripts/harvestapi

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```

Comment:
is this really /bin/sh syntax and not /bin/bash?

Reply:
Ah you young guys. We've been putting #!/bin/sh at the top of our Sys V init scripts since the dawn of time!

But seriously we probably just inherited /bin/sh from wherever Mike found the init script.

If you look at the stats, only 5/62 init scripts on our Ubuntu 14 VMs are Bash, the rest are sh.  Anywho....

Reply:
:)